packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -222,6 +238,9 @@ export default {
         hasImageTags () {
             return !!this.tags?.imageTags?.length;
         },
+        hasOffer () {
+            return !!this.offer?.length || false;
```

Comment:
Don't think you need the `|| false` part here ase the `!!` operators should do this already?

Reply:
Oh yeah, not quite sure why I added that